api/authorizers/site.js

```diff
@@ -39,6 +40,23 @@ const authorizeAdmin = (user, site) => (
     })
 );
 
+const authorizeFederalistUsersAdmin = (user) =>
+  FederalistUsersHelper.federalistUsersAdmins(user.githubAccessToken)
+    .then((admins) => {
+      if (!admins.includes(user.username)) {
+        return Promise.reject({
```

Comment:
If we want the promise to reject just `throw`, there is no need to call `Promise.reject`. In general, when we `throw` we should only be throwing instances of `Error` (or subclasses), but if this works with just the object that's fine for now.